backup-stores/gcs/src/main/java/io/camunda/zeebe/backup/gcs/FileSetManager.java

```diff
@@ -11,26 +11,30 @@
 import com.google.cloud.storage.BucketInfo;
 import com.google.cloud.storage.Storage;
 import com.google.cloud.storage.Storage.BlobWriteOption;
+import io.camunda.zeebe.backup.api.BackupIdentifier;
 import io.camunda.zeebe.backup.api.NamedFileSet;
 import java.io.IOException;
 import java.io.UncheckedIOException;
 
 final class FileSetManager {
+  private static final String FILESET_PREFIX = "contents/";
   private final Storage client;
   private final BucketInfo bucketInfo;
+  private final String prefix;
 
-  FileSetManager(final Storage client, final BucketInfo bucketInfo) {
+  FileSetManager(final Storage client, final BucketInfo bucketInfo, final String prefix) {
```

Comment:
:wrench: Lets rename it to Basepath or something similar, which makes it clear what the intention is